ansible/roles/bind/files/usr/local/sbin/debops-bind-snapshot

```diff
@@ -28,7 +28,7 @@ BACKUP_USER="backup"
 BACKUP_GROUP="backup"
 BACKUP_ROOT="$(getent passwd "${BACKUP_USER}" | cut -f6 -d:)/bind"
 FROZEN_FLAG="/var/lib/bind/.debops.frozen"
-TAR="/usr/bin/tar"
+TAR="/bin/tar"
```

Comment:
BTW, `/bin` in recent Debian releases is a symlink to `/usr/bin` so that technically shouldn't be needed.